webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/yaml/YamlTechnique.scala

```diff
@@ -108,17 +108,18 @@ class YamlTechniqueSerializer(parameterTypeService: ParameterTypeService, resour
     JsonEncoder[String].contramap(s => parameterTypeService.value(s).getOrElse(s.toString))
   implicit val encoderTechniqueParameter:     JsonEncoder[TechniqueParameter] = DeriveJsonEncoder.gen
   implicit val encoderTechnique:              JsonEncoder[Technique]          = DeriveJsonEncoder.gen
```

Comment:
I think we must sort tags when encoding so that the yaml is always written with tags in the same order (easier for git diff and an user looking at the file). 
I'm not sure how it's done  thought - temporaly map to a `SortedMap` and encode that ? Directly use a `SortedMap` for tags ?